server/src/main/resources/schema/wildfly-config_19_0.xsd

```diff
@@ -3815,231 +3815,18 @@
-                    <xs:enumeration value="WildFly22.0">
-                        <xs:annotation>
-                            <xs:documentation>
-                                <![CDATA[
-                                   A shorthand identifier for kernel management API version 15.0.x
-                                ]]>
-                            </xs:documentation>
-                        </xs:annotation>
-                    </xs:enumeration>
-                    <xs:enumeration value="EAP7.4">
-                        <xs:annotation>
-                            <xs:documentation>
-                                <![CDATA[
-                                   A shorthand identifier for kernel management API version 16.0.x
-                                ]]>
-                            </xs:documentation>
-                        </xs:annotation>
-                    </xs:enumeration>
-                    <xs:enumeration value="WildFly23.0">
-                        <xs:annotation>
-                            <xs:documentation>
-                                <![CDATA[
-                                   A shorthand identifier for kernel management API version 16.0.x
-                                ]]>
-                            </xs:documentation>
-                        </xs:annotation>
-                    </xs:enumeration>
-                    <xs:enumeration value="WildFly24.0">
-                        <xs:annotation>
-                            <xs:documentation>
-                                <![CDATA[
-                                   A shorthand identifier for kernel management API version 17.0.x
-                                ]]>
-                            </xs:documentation>
-                        </xs:annotation>
-                    </xs:enumeration>
-                    <xs:enumeration value="WildFly25.0">
-                        <xs:annotation>
-                            <xs:documentation>
-                                <![CDATA[
-                                   A shorthand identifier for kernel management API version 18.0.x
-                                ]]>
-                            </xs:documentation>
-                        </xs:annotation>
-                    </xs:enumeration>
+                    <xs:annotation>
+                        <xs:documentation>
+                            <![CDATA[
+                              The short hand identifier can have to form:
+                              * EAP<major>.<minor> (for example, EAP6.2 or EAP7.3)
+                              * WildFly<major>.<minor> (for example WildFly10.0 or WildFly10.1)
+
+                              The corresponding kernel management API version is defined in the enum
+                              org.jboss.as.domain.controller.resources.HostExcludeResourceDefinition.KnownRelease.
+                            ]]>
+                        </xs:documentation>
+                    </xs:annotation>
```

Comment:
This change looks good, this is rejected by the attribute validator and not by the XSD schema.